### ISSUE TITLE
Can I register different custom datasets with the same "dataset name"?

### ISSUE THREAD

**Issue by Zy1013921842** [facebookresearch/detectron2#473]
Since we may trian a lot of times, if the custom dataset name "my_dataset" can be registerd with different image paths and annotation paths again and again, it will be much more convenient for users to avoid giving them too much names!

Can this be achieved?

**Comment by ppwwyyxx** [comment]
It's not supported and will likely not. I cannot understand what your use case is. It would make more sense if you give a more concrete example.

**Comment by Zy1013921842** [comment]
> I cannot understand what your use case is or what exactly you're asking. It would make more sense if you give a more concrete example of what you'd like to do.

Hi, friend.
Imagine this, I have registered a custom dataset named "my_dataset", and now I want to add more images to the same directory as registered in "my_dataset" , along with a new json file. Since we should give the full path including file name of json, once json is changed,  we should create a new custom dataset name to register again, which is not convenient if we train quiet often. Am I clear? Or this can be done and I made any mistakes. 
Thanks for your reply!
